plugins/autopeering/local/local.go

```diff
@@ -26,41 +26,55 @@ var (
 func configureLocal() *peer.Local {
 	log := logger.NewLogger("Local")
 
-	var externalIP net.IP
+	var peeringIP net.IP
 	if str := config.NodeConfig.GetString(config.CfgNetAutopeeringExternalAddr); strings.ToLower(str) == "auto" {
-		log.Info("Querying external IP ...")
-		ip, err := netutil.GetPublicIP(config.NodeConfig.GetBool(config.CfgNetPreferIPv6))
-		if err != nil {
-			log.Fatalf("Error querying external IP: %s", err)
-		}
-		log.Infof("External IP queried: address=%s", ip.String())
-		externalIP = ip
+		// let the autopeering discover the IP
+		peeringIP = net.IPv4zero
```

Comment:
It is just cosmetic, but maybe consider using `net.IPv6unspecified` when IP v6 is preferred.
But this wont have any impact on the behavior, e.g. the pong will always be send to the source IP of the ping, using IPv4 is the source is v4 and IPv6 if the source is v6.